symfony/client/src/orangehrmPimPlugin/pages/dataImport/EmployeeDataImport.vue

```diff
@@ -0,0 +1,204 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-card-container">
+      <oxd-text class="orangehrm-main-title">Data Import</oxd-text>
+
+      <oxd-divider />
+      <div class="orangehrm-information-card-container">
+        <oxd-text class="orangehrm-sub-title">Note:</oxd-text>
+        <ul>
+          <li>
+            <oxd-text class="orangehrm-information-card-text"
+              >Column order should not be changed</oxd-text
+            >
+          </li>
+          <li>
+            <oxd-text class="orangehrm-information-card-text"
+              >First Name and Last Name are compulsory</oxd-text
+            >
+          </li>
+          <li>
+            <oxd-text class="orangehrm-information-card-text"
+              >All date fields should be in YYYY-MM-DD format</oxd-text
+            >
+          </li>
+          <li>
+            <oxd-text class="orangehrm-information-card-text"
+              >If gender is specified, value should be either Male or
+              Female</oxd-text
+            >
+          </li>
+          <li>
+            <oxd-text class="orangehrm-information-card-text"
+              >Each import file should be configured for 100 records or
+              less</oxd-text
+            >
+          </li>
+          <li>
+            <oxd-text class="orangehrm-information-card-text"
+              >Multiple import files may be required</oxd-text
+            >
+          </li>
+          <li>
+            <oxd-text class="orangehrm-information-card-text"
+              >Sample CSV file:
+              <a class="download-link" href="#">
+                <span @click="onClickDownload" class="download-link"
```

Comment:
Here it can be either anchor tag or span, feels like not need both. if using anchor tag, prevent modifier is preferred as `@click.prevent`